src/types/api/featureFlags.ts

```diff
@@ -0,0 +1,14 @@
+export interface TConfigFeatureFlag {
+    Name: string;
+    Current?: boolean;
+    Default?: boolean;
+}
+
+export interface TConfigDb {
+    Name: string;
+    FeatureFlags: TConfigFeatureFlag[];
+}
+
+export interface TConfigs {
+    Databases: TConfigDb[];
+}
```

Comment:
Not a request, but for you information: types are called TSomething and ESomething to match with names from Swagger or backend types, you may not follow the same pattern for your own types